### PR TITLE
Docker branch

### PR DESCRIPTION
**Pull request by SalnikovMarkiian**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,9 @@
+MYSQLDB_USER=root
+MYSQLDB_ROOT_PASSWORD=1234
+MYSQLDB_DATABASE=online_book_store
+MYSQLDB_LOCAL_PORT=3307
+MYSQLDB_DOCKER_PORT=3306
+
+SPRING_LOCAL_PORT=8088
+SPRING_DOCKER_PORT=8080
+DEBUG_PORT=5005
```
Comment (teract10s) [inline .env]: You haven't to push it to Git Hub. You may delete the file, push, and add it again to a local folder. Maybe you added ".env" to .gitignore after git detects this file.

Comment (teract10s) [reply .env]: Also, you may use these variables in application.properites using ${var_name}

**Review by teract10s — CHANGES_REQUESTED** [review]
Good job
Just minor comments

File: pom.xml [hunk 2]
```diff
@@ -99,8 +112,15 @@
             <version>${jjwt.version}</version>
             <scope>runtime</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-docker-compose</artifactId>
```
Comment (NazarSmith228) [inline pom.xml:118]: why do you need this?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+COPY ${JAR_FILE} online-book-store.jar
+RUN java -Djarmode=layertools -jar online-book-store.jar extract
+
+# Final stage
+FROM openjdk:17-jdk-alpine
+WORKDIR application
+COPY --from=builder online-book-store/dependencies/ ./
+COPY --from=builder online-book-store/spring-boot-loader/ ./
+COPY --from=builder online-book-store/snapshot-dependencies/ ./
+COPY --from=builder online-book-store/application/ ./
```
Comment (NazarSmith228) [inline Dockerfile:14]: I don't see any `application` folder at builder stage

or it is one of default folders after jar extraction?

File: docker-compose.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            "bookstore.openapi.dev-url" : "http://localhost:$SPRING_LOCAL_PORT"
+          }'
+      JAVA_TOOL_OPTIONS: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
+  book-store-db:
+    container_name: book-store-db
+    image: mysql
+    restart: unless-stopped
+    env_file: ./.env
+    environment:
+      - MySQL_DATABASE=$MYSQLDB_DATABASE
```
Comment (NazarSmith228) [inline docker-compose.yaml:29]: should it be `MySQL` or `MYSQL` ? 

just seems like a typo

**Review by NazarSmith228 — APPROVED** [review]
LGTM, but pls pay attention to my comments

**Pull request merged**